modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,49 @@
+= Share an Asset
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+:page-aliases: to-share-an-asset-with-a-user.adoc
+
+Exchange administrators and asset administrators can share an asset with another user in their organization or with an external organization.
+
+== Share With a User
+
+If the person is not in your organization,
+click Access Management > Users to add them.
+
+. Click an asset to show its details and click Share.
+. In the Share menu, type a few letters of the person's name you want to share this asset with.
+Or click a filter to list only users who have Viewer, Contributor, or Admin rights.
+. Click Add.
+. Repeat steps 2 and 3 to add additional users to share the asset with.
+
+You can also use the Share menu to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With an External Organization
+
+Exchange administrators and asset administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-org-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. Login to your Exchange administrator account in the granting organization.
+. Click an asset to show its details and click Share.
+. In the Share menu, begin typing the name of the external organization you want to share the asset with.
+. Select the organization you want to grant viewer access to.
+. Repeat steps 4 and 5 to add additional organizations to share the asset with.
+
+When an asset is shared with an organization, email notification is sent to the maintainer of the organization's default "All Users" team. Usually this is the organization administrator.
```

Comment:
The email is sent to `organization administrators`

Reply:
Thanks! Fixed.